docs/core/diagnostics/dotnet-counters.md

```diff
@@ -430,8 +430,8 @@ dotnet-counters ps [-h|--help]
 ```console
 > dotnet-counters ps
   
-  15683 WebApi     /home/user/repos/WebApi/WebApi
-  16324 dotnet     /usr/local/share/dotnet/dotnet
+  15683 WebApi     /home/user/repos/WebApi/WebApi   commandline arguments 1
+  16324 dotnet     /usr/local/share/dotnet/dotnet   commandline arguments 2
```

Comment:
Maybe it's better to first show how these processes were started, and then show list the actual output from `dotnet-counters ps`.

Also on line 420, can you add: "On .NET 7 and later versions, also displays the command-line arguments that each process was started with."